staging/src/github.com/openshift/openshift-controller-manager/pkg/template/controller/templateinstance_controller.go

```diff
@@ -345,6 +345,21 @@ func (c *TemplateInstanceController) enqueueAfter(templateInstance *templatev1.T
 	c.queue.AddAfter(key, duration)
 }
 
+func (c *TemplateInstanceController) processNamespace(templateNamespace, objNamespace string, clusterScoped bool) (string, error) {
+	namespace := ""
+	var err error
+	if !clusterScoped {
```

Comment:
nit: by flipping the `if` you make this less nested `if clusterScoped { return "", nil }`

Reply:
update pushed